desktop/apps/auction/test/routes.js

```diff
@@ -102,7 +102,7 @@ describe('#redirectLive', () => {
     RoutesRewireApi.__Rewire__('metaphysics', sinon.stub().returns(Promise.resolve(auctionQueries)))
     res = {
       redirect: (url) => {
-        url.should.equal('https://live.artsy.net/foo/login')
+        url.should.equal(`${process.env.PREDICTION_URL}/foo/login`)
```

Comment:
As far as I can tell, sharify isn't available yet in this express handler. I changed the original function to fall back to process.env following the comment here:
https://github.com/erikdstock/force/blob/auctions-420-live-links-login/config.js#L4